ml3d/tf/models/network_blocks.py

```diff
@@ -82,7 +82,7 @@ def block_decider(block_name, radius, in_dim, out_dim, layer_ind, cfg):
                           out_dim,
                           cfg.use_batch_norm,
                           cfg.batch_norm_momentum,
-                          l_relu=cfg.get('l_relu', 0.1))
```

Comment:
as far as i remember, 0.2 was for converted weights from toronto3d and 0.1 was for all other datasets trained using our pipeline. Will it still work?

Reply:
The default slope rate is 0.2 in the original code. (https://github.com/HuguesTHOMAS/KPConv/blob/07861d8e40244c20ede5a362052e01f61dcc012a/models/network_blocks.py#L166)

I will also run the inference for all the datasets again.

Reply:
The default slope rate is 0.2 in the original code. (https://github.com/HuguesTHOMAS/KPConv/blob/07861d8e40244c20ede5a362052e01f61dcc012a/models/network_blocks.py#L166)

I will also run the inference for all the datasets again.